integrationtest/performance/orm/src/test/java/org/hibernate/search/test/performance/scenario/TestContext.java

```diff
@@ -41,20 +41,20 @@
 	public TestContext(SessionFactory sessionFactory) {
 		this.sessionFactory = sessionFactory;
 
-		if ( PERFORMANCE_ENABLED ) {
+		if ( performanceEnabled ) {
 			this.initialOffset = 1_000_000;
 			this.initialAuthorCount = 10_000;
 			this.initialBookCount = 1_000_000;
-			this.warmupCyclesCount = 1_000;
-			this.measuredCyclesCount = 5_000;
 		}
 		else {
 			this.initialOffset = 100;
 			this.initialAuthorCount = 10;
 			this.initialBookCount = 100;
-			this.warmupCyclesCount = 1;
-			this.measuredCyclesCount = 1;
 		}
+
+		this.threadCount = 2;
+		this.warmupCyclesCount = 1;
```

Comment:
This looks like suspicious. Just one cycle for warming up the JVM even when performance tests are enabled?

Reply:
I moved this configuration to the tests themselves, because it makes no sense to say "hey, let's do a thousand warmup cycles" when we don't know how expensive one cycle is. I can guarantee that one thousand mass indexing cycles are way too much.

See `org.hibernate.search.test.performance.scenario.ReadWriteTestScenario#initContext`

Reply:
oh, ok. Add a comment here for future understanding?